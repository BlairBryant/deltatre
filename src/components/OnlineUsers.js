import React from 'react'


function OnlineUsers({toggle, users}) {
  console.log(toggle)
  return (
    <div className={toggle ? "users users--open" : "users"}>
      <h2 className='users__title'>Online Users: {users.length}</h2>
      {
        users.map(user => (
          <div className='user' key={user.id}>
            <div className='user__container'>
              <img className='user__img' src={user.img} alt='User'/>
              <span className='user__name'>{user.name}</span>
            </div>
            <hr className='user__hr'/>
          </div>
        ))
      }
    </div>
  )
}

export default OnlineUsers