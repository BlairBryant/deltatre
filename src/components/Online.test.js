import React from "react";
import renderer from 'react-test-renderer'
import OnlineUsers from "./OnlineUsers";

test('Online Users', () => {
    const component = renderer.create(
      <OnlineUsers toggle={true} users={[]}/>
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()

    
})