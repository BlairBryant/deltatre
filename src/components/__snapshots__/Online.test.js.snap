// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Online Users 1`] = `[Function]`;
